pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -66,25 +74,29 @@ let
     sha256 = "sha256-L1B5Y/c897Jg9fGwT2J3+vaXsZ+lfXnskp8Gto1p/Tg=";
   };
 
-  gtest' = gtest.overrideAttrs (oldAttrs: rec {
-    version = "1.13.0";
-    src = fetchFromGitHub {
-      owner = "google";
-      repo = "googletest";
-      rev = "v${version}";
-      hash = "sha256-LVLEn+e7c8013pwiLzJiiIObyrlbBHYaioO/SWbItPQ=";
-    };
-    });
+  gtest = fetchFromGitHub {
+    owner = "google";
+    repo = "googletest";
+    rev = "v1.14.0";
+    hash = "sha256-t0RchAHTJbuI5YW4uyBPykTvcjy90JW9AOPNjIhwh6U=";
+  };
+
+  onnx = fetchFromGitHub {
+    owner = "onnx";
+    repo = "onnx";
+    rev = "refs/tags/v1.14.1";
```

Comment:
Should onnx dependency version be also updated with onnxruntime?

Reply:
The dependency is pinned upstream. 1.16.3 uses ONNX version 1.14.1.

Reply:
Ok

Reply:
Do you mind adding a comment where to check for runtime -> onnx versions

Reply:
Pinned dependencies are found here: https://github.com/microsoft/onnxruntime/blob/main/cmake/deps.txt#L38